src/dotnet/install.sh

```diff
@@ -4,489 +4,133 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 #
-# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/dotnet.md
+# Docs: https://github.com/devcontainers/features/tree/main/src/dotnet
 # Maintainer: The VS Code and Codespaces Teams
-
 DOTNET_VERSION="${VERSION:-"latest"}"
-DOTNET_RUNTIME_ONLY="${RUNTIMEONLY:-"false"}"
-OVERRIDE_DEFAULT_VERSION="${OVERRIDEDEFAULTVERSION:-"true"}"
-INSTALL_USING_APT="${INSTALLUSINGAPT:-"true"}"
-
-DOTNET_LATEST="7"
-DOTNET_LTS="6"
-
-USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
-UPDATE_RC="${UPDATE_RC:-"true"}"
-TARGET_DOTNET_ROOT="${TARGET_DOTNET_ROOT:-"/usr/local/dotnet"}"
-ACCESS_GROUP="${ACCESS_GROUP:-"dotnet"}"
-
-MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
-DOTNET_ARCHIVE_ARCHITECTURES="amd64"
-DOTNET_ARCHIVE_VERSION_CODENAMES="buster bullseye bionic focal hirsute jammy"
-# Feed URI sourced from the official dotnet-install.sh
-# https://github.com/dotnet/install-scripts/blob/1b98b94a6f6d81cc4845eb88e0195fac67caa0a6/src/dotnet-install.sh#L1342-L1343
-DOTNET_CDN_FEED_URI="https://dotnetcli.azureedge.net"
-# Ubuntu 22.04 and on do not ship with libssl1.1, which is required for versions of .NET < 6.0
-DOTNET_VERSION_CODENAMES_REQUIRE_OLDER_LIBSSL_1="buster bullseye bionic focal hirsute"
-
-# Comma-separated list of dotnet versions to be installed
-# alongside DOTNET_VERSION, but not set as default.
-ADDITIONAL_VERSIONS=${ADDITIONALVERSIONS:-""}
-
-APT_PREFERENCES_CONTENT="$(cat << 'EOF'
-Package: *
-Pin: origin "packages.microsoft.com"
-Pin-Priority: 1001
-EOF
-)"
+DOTNET_ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-APT_PREFERENCES_FILE_ADDED="false"
-APT_PREFERENCES_UPDATED="false"
+DOTNET_INSTALL_SCRIPT_URL='https://dot.net/v1/dotnet-install.sh'
```

Comment:
I've never done this but I imagine a GitHub workflow could fetch the latest script and create a pull request if there are any changes?

Reply:
Can you add a simple script that downloads the latest version? For example, `src/dotnet/scripts/fetch-latest-dotnet-install.sh`

```bash
pushd src/dotnet/scripts
wget https://dot.net/v1/dotnet-install.sh -O dotnet-install.sh
```

The `scripts` directory can hold `dotnet-install.sh` and `fetch-latest-dotnet-install.sh`

We can run it manually for now, or add a simple automation in the future.

Reply:
@jungaretti done in 52008539b5ffcaf8fcee7314d5bf30103a676aa2

Reply:
💡 @sliekens Would you be interested in adding the workflow file? It would be very similar to https://github.com/devcontainers/features/blob/main/.github/workflows/update-documentation.yml

Instead of the `devcontainers/action`, you could use the commands from [fetch-latest-dotnet-install.sh](https://github.com/devcontainers/features/blob/e39d87bb202a1704e5d15377be14804479e2aebc/src/dotnet/scripts/fetch-latest-dotnet-install.sh) to open a PR (if there are any updates to `dotnet-install.sh`). I can provide help with the changes!

Reply:
@samruddhikhandale  I can try but how would I test it? Run it against my fork?

Reply:
> Run it against my fork?

Yes. However, feel free to contribute the change in a separate PR,. I don't think it's necessary/needed for this PR to be merged. 👍

Reply:
Done in 37c220fb6cde72ebf5fee46e67d6c47bc0b8165e